### PR TITLE
refactor: small tweaks to structure

### PR DESCRIPTION
**Pull request by cpvalente**
some suggestions for improving the structure. mostly simplifying logic

### CODE REVIEW

File: apps/server/src/services/PlaybackService.ts [hunk 1]
```diff
@@ -242,30 +243,33 @@ export class PlaybackService {
...
+      return;
+    }
+
+    if (restorePoint.playback === Playback.Pause || restorePoint.playback === Playback.Play) {
+      const event = EventLoader.getEventWithId(restorePoint.selectedEventId);
+      const success = PlaybackService.loadEvent(event);
+      if (success) {
+        willResume();
+        // event timer already got the event from the playbackService
+        // TODO: this will cause a flash of outdated data
```
Comment (cpvalente) [inline apps/server/src/services/PlaybackService.ts:264]: This is a small thing, but we can improve

File: apps/server/src/services/PlaybackService.ts [hunk 2]
```diff
@@ -242,30 +243,33 @@ export class PlaybackService {
...
+        eventTimer.resume(restorePoint.startedAt, restorePoint.addedTime, restorePoint.pausedAt);
```
Comment (alex-Arc) [inline apps/server/src/services/PlaybackService.ts:265]: we need to passe the `restorePoint.playback` to eventTimer

File: apps/server/src/services/TimerService.ts [hunk 3]
```diff
@@ -78,47 +87,16 @@ export class TimerService {
...
    * @param {number} addedTime
    * @param {number} pausedAt
    */
-  resume(timer, playback: Playback, selectedEventId: string | null, startedAt: number | null, addedTime: number | null, pausedAt: number | null) {
-    if (typeof timer === 'undefined') {
-      this.stop();
-      return;
-    }
-    this.timer.selectedEventId = selectedEventId;
+  resume(startedAt: number | null, addedTime: number | null, pausedAt: number | null) {
```
Comment (alex-Arc) [inline apps/server/src/services/TimerService.ts:94]: we need to recive the `restorePoint.playback` from PlaybackService

**Review by alex-Arc — APPROVED** [review]
looks good and works  for my test

**Pull request merged**